whenchanged/whenchanged.py

```diff
@@ -191,7 +206,17 @@ def main():
         if verbose:
             print("When '%s' changes, run '%s'" % (files[0], print_command))
 
-    wc = WhenChanged(files, command, recursive, run_once, run_at_start)
+    return {"files":files, "command":command, "recursive":recursive, "run_once":run_once, "run_at_start":run_at_start}
+
+def main():
+    try:
+        wc = WhenChanged(**parse_args(sys.argv))
```

Comment:
parse_args can return None, so need to check this before passing to WhenChanged()